app/assets/javascripts/discourse/app/lib/sidebar/custom-sections.js

```diff
@@ -1,14 +1,54 @@
 import BaseCustomSidebarSection from "discourse/lib/sidebar/base-custom-sidebar-section";
+import BaseCustomSidebarPanel from "discourse/lib/sidebar/base-custom-sidebar-panel";
 import BaseCustomSidebarSectionLink from "discourse/lib/sidebar/base-custom-sidebar-section-link";
+import I18n from "I18n";
 
-export const customSections = [];
+class MainSidebarPanel {
+  sections = [];
 
-export function addSidebarSection(func) {
-  customSections.push(
+  get key() {
+    return "main";
+  }
+  get switchButtonLabel() {
+    return I18n.t("sidebar.panels.forum.label");
+  }
+
+  get switchButtonIcon() {
+    return "random";
+  }
+
+  get switchButtonDefaultUrl() {
+    return "/latest";
+  }
+}
+
+export let customPanels = [new MainSidebarPanel()];
```

Comment:
```suggestion
export const customPanels = [new MainSidebarPanel()];
```

Reply:
I just ensured, but this cannot be `const` as reset is reassigning it.

Reply:
ahhh icic :+1: